binaries/scheduler/main.go

```diff
@@ -24,6 +30,14 @@ func main() {
 	// TODO: Replace with Durable WorkQueue, currently in Memory Only
 	workQueue := memory.NewSimpleQueue(1000)
 
+	handler := server.NewHandler(workQueue, sagaCoordinator)
+
+	// Start API Server
+	err := server.Serve(handler, addr, transportFactory, protocolFactory)
+	if err != nil {
+		log.Fatal("Error serving Scoot API: ", err)
+	}
+
 	var wg sync.WaitGroup
 	wg.Add(1)
 
```

Comment:
Reading this, it seems like it won't actually get to the work generator, because the Serve will wait. Am I missing something?